### PR TITLE
Separate CAS host and URL for the Omniauth config.

### LINKED ISSUE #265: Builds fail for user authentication with Net::ReadTimeout
In some cases, test builds fail on CircleCI with the following message:
```
1) Ephemera Terms when users have added a controlled vocabulary when users have added terms to the vocabulary users can edit existing terms
     Failure/Error: visit user_cas_omniauth_authorize_path
     
     Net::ReadTimeout:
       Net::ReadTimeout
     # ./spec/support/features/session_helpers.rb:12:in `sign_in'
     # ./spec/features/ephemera_terms_spec.rb:13:in `block (2 levels) in <top (required)>'
```

### PR DESCRIPTION
**Pull request by jrgriffiniii**
This attempts to resolve #265

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -252,7 +252,9 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :cas, host: 'fed.princeton.edu', url: 'https://fed.princeton.edu/cas'
+  config.omniauth(:cas,
```
Comment (tpendragon) [inline config/initializers/devise.rb]: Can we just put these in config.yml?

Comment (jrgriffiniii) [reply config/initializers/devise.rb]: Agreed, I initially attempted this (there was some conflict in the loading procedure), but wanted to get the branch out for testing on Circle first

**Comment by jrgriffiniii** [comment]
This has been updated with the identified improvements and is prepared for a second review.

**Comment by jrgriffiniii** [comment]
This has been updated with the identified improvements, further tested, and is now prepared for a second review.

**Comment by jrgriffiniii** [comment]
The bug was once again reproduced (perhaps another environment was used?)

**Comment by escowles** [comment]
@jrgriffiniii dang, that seemed really promising 😡

**Comment by escowles** [comment]
I wonder if this might work to resolve this issue? https://stackoverflow.com/a/39666664

**Comment by jrgriffiniii** [comment]
Given the urgency of this issue and my failure to reproduce this after 15 rebuilds, I have submitted this for review.

**Pull request merged**